static/bidder-params/yieldone.json

```diff
@@ -0,0 +1,15 @@
+
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Yieldone Adapter Params",
+    "description": "A schema which validates params accepted by the Yieldone adapter",
+  
+    "type": "object",
+    "properties": {
+      "placementId": {
```

Comment:
The `placementId` param is not being used by this adapter anywhere. See comments left in `adapters/yieldone/yieldone.go`